### PR TITLE
[monica] initial add for monica

### PR DESCRIPTION
**Pull request by carpenike**
Signed-off-by: Ryan Holt <ryan@ryanholt.net>

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

File: charts/monica/values.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  tag: 2.19.1-apache
+
+strategy:
+  type: Recreate
+
+service:
+  port:
+    port: 80
+
+env:
```
Comment (onedr0p) [inline charts/monica/values.yaml:15]: `env: {}`

File: charts/monica/values.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  # See the Monica documentation for all available options for environment variables: https://raw.githubusercontent.com/monicahq/monica/master/.env.example
+  APP_ENV: production
```
Comment (onedr0p) [inline charts/monica/values.yaml:17]: Comment out all these environment variables

File: charts/monica/values.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+persistence:
+  config:
+    enabled: false
+    emptyDir: false
+    mountPath: /var/www/html/storage
+
+# Enabled mariadb
+# ... for more options see https://github.com/bitnami/charts/tree/master/bitnami/mariadb
+mariadb:
+  enabled: true
```
Comment (onedr0p) [inline charts/monica/values.yaml:35]: set to false

**Review by onedr0p — CHANGES_REQUESTED** [review]

File: charts/monica/values.yaml (hunk 2 above)
Comment (onedr0p) [reply charts/monica/values.yaml:17]: and add to ct-values.yaml

File: charts/monica/values.yaml (hunk 3 above)
Comment (onedr0p) [reply charts/monica/values.yaml:35]: and add to ct-values.yaml

Comment (carpenike) [reply charts/monica/values.yaml:35]: should be done.

File: charts/monica/values.yaml (hunk 2 above)
Comment (carpenike) [reply charts/monica/values.yaml:17]: done.

File: charts/monica/values.yaml (hunk 1 above)
Comment (carpenike) [reply charts/monica/values.yaml:15]: done.

**Review by onedr0p — APPROVED** [review]

**Pull request merged**